### DISCUSSION TITLE
Clockwork Beta 15 - Release announcement

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#64] [Announcements]
Hi there! We're releasing Clockwork 2.0.0.0-beta15, and with it a slew of new features! 
- We've been working on accessibility for users, and that includes the addition of a dyslexia-friendly font toggle! It only applies to certain parts of Clockwork right now, but we're working to get it applied to the rest of the UI (and maybe even apps!)
- A new wallpaper has been added to celebrate 100 stars: Starry!
![starry](https://github.com/red-stone-network/clockwork/assets/91223726/029bb656-b8ac-45fa-9c64-313a89e9f740)
- You can now import data and factory reset Clockwork from the Settings app!
- New users will now see a new setup screen when first booting into Clockwork!
- And the latest addition for developers: CDN URLs! You can now use a URL like `@gh/username/repo@branch/path/to/your/file` as the app URL, which will get the app's JSON file from an unblocked Github/Gitlab CDN. You should try to use these when you can - they're so much better for the end user.

This update should be out to all instances by the end of the day! There are way more changes than we mention here - you can find the update logs [here.](https://github.com/red-stone-network/clockwork/blob/main/UPDATELOGS.md)

**Comment by Ender-Ghost-Craft** [comment]
can you change the store app cuz its blocked

**Reply by ToasterPanic** [reply]
[Use a proxy](https://github.com/red-stone-network/clockwork/wiki/User-Guide#use-an-ultraviolet-proxy-with-clockwork-apps)

**Reply by Ender-Ghost-Craft** [reply]
how do I add the url? the code is 
self.__uv$config = {
    prefix: '/school/',
    bare: '/bare/',
    encodeUrl: Ultraviolet.codec.xor.encode,
    decodeUrl: Ultraviolet.codec.xor.decode,
    handler: '/uv/uv.handler.js',
    bundle: '/uv/uv.bundle.js',
    config: '/uv/uv.config.js',
    sw: '/uv/uv.sw.js',
    da: 'a'
};

**Reply by Ender-Ghost-Craft** [reply]
what should I put into the proxy link with this code? the URL to the proxy is 69.eightgrade.com

**Reply by Ender-Ghost-Craft** [reply]
how do I add the url? the code is
self.__uv$config = {
prefix: '/school/',
bare: '/bare/',
encodeUrl: Ultraviolet.codec.xor.encode,
decodeUrl: Ultraviolet.codec.xor.decode,
handler: '/uv/uv.handler.js',
bundle: '/uv/uv.bundle.js',
config: '/uv/uv.config.js',
sw: '/uv/uv.sw.js',
da: 'a'
};

and the proxy is 69.eightgrade.com/

**Reply by ToasterPanic** [reply]
can't rememer if i've responded to you yet, `https://69.eightgrade.com/school/`
